### PR TITLE
Completed route for GET photos on /api/home/:id/photos

### PR DESCRIPTION
**Pull request by kcords**
Ready for review

### CODE REVIEW

**Comment by kcords** [comment]
[Create routes for API requests](https://trello.com/c/ogL1bJ1Z/13-create-routes-for-api-requests)

File: public/bundle.js [hunk 1]
```diff
@@ -7,9 +7,6 @@
  * If you are looking for production-ready output files, see mode: "production" (https://webpack.js.org/configuration/mode/).
  */
 /******/ (() => { // webpackBootstrap
-/*!******************************!*\
-  !*** ./client/src/index.jsx ***!
-  \******************************/
-eval("throw new Error(\"Module parse failed: Unexpected token (5:2)\\nYou may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders\\n| \\n| ReactDOM.render(\\n>   <h1>Hello World!</h1>,\\n|   document.getElementById('photos-module'),\\n| );\");\n\n//# sourceURL=webpack://photo-carousel-service/./client/src/index.jsx?");
+/******/ 	"use strict";
```
Comment (juanoxmar) [inline public/bundle.js:10]: add your bundle.js to your .gitignore

**Review by juanoxmar — CHANGES_REQUESTED** [review]
Only one thing.

**Review by apat2010 — COMMENTED** [review]
Looks great!

File: public/bundle.js (hunk 1 above)
Comment (kcords) [reply public/bundle.js:10]: bundle.js has been added to gitignore.

**Review by apat2010 — APPROVED** [review]
Reviewed! Looks clean and addressed Juan's concern

**Pull request merged**